ProjectSettings/ProjectSettings.asset

```diff
@@ -668,7 +668,9 @@ PlayerSettings:
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
     1: 
-    14: 
+    4: ARFOUNDATION_PRESENT
```

Comment:
We don't want to commit these defines, but let the code resolve them when the project is loaded in the editor.

Reply:
Otherwise, looks good!

Reply:
Ok, they are removed.